crates/nu-cmd-dataframe/src/dataframe/values/nu_expression/mod.rs

```diff
@@ -433,5 +426,8 @@ pub fn expr_to_value(expr: &Expr, span: Span) -> Value {
                 span,
             )
         }
+        // Selector is part of the polars_plan package. It is currently marked as nightly.
+        // Wasn't sure what the best approach to take here would be.
+        Expr::Selector(_) => todo!("Add selector support"),
```

Comment:
I believe we should use some type of `ShellError::UnsupportedInput` or create a NonImpleletedYet versus having a `todo!`.

Reply:
I'll change.. I dug in a little more, my comment here is incorrect and updated it.  It looks like polars_plan::dsl::selector::Selector isn't publicly exposed, so we simply can't do anything with it directly. I'll dig through polars a bit more to see if there is anything I can do with the reference. If not I will change to UnsupportedInput.